### ISSUE TITLE
Add bakery command to print all routes

### ISSUE THREAD

**Issue by apple314159** [userfrosting/UserFrosting#902]
To improve project documentation and facilitate the creation of test cases, it would be very useful to have an option to dump all the project routes. The output could be something like this:

```
$ php bakery routes
GET /account/captcha
GET /account/check-username
…
POST /account/login
...
```

**Comment by lcharette** [comment]
Will requires #854 to be fixed first.

**Comment by lcharette** [comment]
Implemented in 4.1.20 through PR #903 and improved for 4.2 in 978db3768920cc9e835801ede4a0bbd7bd3ecba8. Closing !
